### PR TITLE
WS2-1679: Review all components for being in sync with UDS and Bootstrap 5

### PR DESCRIPTION
**Pull request by bebaps**
**This PR includes changes to the file structure of the theme CSS files and other files. Do not merge this PR unless you have reviewed the changes locally and approve of them.**

---

This PR includes the updates to the components included in WS2 to ensure that the markup, styles, and scripts match what is currently in use by the most recent version of UDS. This PR also includes changes to WS that would affect developers in the following ways:

- The CSS build now pulls directly from the UDS package, ensuring that there is no longer a need to manually copy the CSS from one folder to another.
- Laravel Mix has been updated to Laravel Mix 6.
- The `webpack.mix.js` file has been updated to allow for pulling and compiling files directly from UDS, and the `config.js` has been simplified.
- Where necessary, CSS has been moved out of the `src/components` folder and into the main `src/sass` folder, with imports being updated accordingly. This was done to keep code organized by scope.
- Some files have had documentation added to them to help developers use them properly.
- Some files have been deleted, as it was found that they were never used to begin with.

**With this PR, it is important to note that it is being opened due to the scope of the JIRA ticket being completed. In the process of this task, issues with the change to Bootstrap 5 have been identified that remain to be addressed. This PR will not resolve those issues, and they must be addressed in subsequent JIRA tickets.**

Due to this, there will not be a standard QA process, as this task simply aims to bring the markup, CSS, and JS in sync with UDS. They only way to confirm this is to manually review the Storybook components for their markup and functionality. We also do not want our QA tester to create new tickets for issues that we already know appeared due to the Bootstrap 5 switchover.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1679)

### Checklist

- Design updates match [Web Standards](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/) and [Unity Design System](https://unity.web.asu.edu)
- Solution is documented on the Jira ticket
- No new PHP or JS errors
- No accessibility issues are introduced with this update
- Added/updated README.md files, if relevant

### Verified in browsers 

- Chrome
- Safari
- Firefox

### CODE REVIEW

**Comment by mlsamuelson** [comment]
[Visit Site](https://pr-142-webspark-ci.pantheonsite.io/)

Created multidev environment [pr-142](https://dashboard.pantheon.io/sites/aec2f75f-35eb-41f8-abe7-95b292415259#pr-142) for webspark-ci.

**Comment by juanmitriatti** [comment]
Hi @bebaps 
I tested it locally and everything looks fine to me.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
Notes on changes requested as discussed.

File: web/themes/webspark/renovation/assets/mix-manifest.json
Comment (bebaps) [reply web/themes/webspark/renovation/assets/mix-manifest.json:6]: Removed the files.

File: web/themes/webspark/renovation/renovation.libraries.yml
Comment (bebaps) [reply web/themes/webspark/renovation/renovation.libraries.yml:7]: Removed the Bootstrap file.

File: web/themes/webspark/renovation/src/sass/pages/_degree-details.scss
Comment (bebaps) [reply web/themes/webspark/renovation/src/sass/pages/_degree-details.scss:1]: Moved the location of the file.

**Review by mlsamuelson — APPROVED** [review]
This looks good. I'm going to wait until tomorrow to merge just to give others a bit more time if they want to chime in, but with approval from Juan Pablo and I, it's vetted.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
I discovered the styles aren’t applying for the node edit tabs. Perhaps https://github.com/ASUWebPlatforms/webspark-ci/pull/142/files#diff-fe509a9f712527f7d3b7748fce570466c4007e413b9da0a2a8b21510db7515f3 is needed after all. Looks like it’s commented out at https://github.com/ASUWebPlatforms/webspark-ci/pull/142/files#diff-087ac902deab617ca34478e84da12f4af076d3098c45c7bb818a974feb3e3067R17

**Comment by bebaps** [comment]
> I discovered the styles aren’t applying for the node edit tabs. Perhaps https://github.com/ASUWebPlatforms/webspark-ci/pull/142/files#diff-fe509a9f712527f7d3b7748fce570466c4007e413b9da0a2a8b21510db7515f3 is needed after all. Looks like it’s commented out at https://github.com/ASUWebPlatforms/webspark-ci/pull/142/files#diff-087ac902deab617ca34478e84da12f4af076d3098c45c7bb818a974feb3e3067R17

@mlsamuelson  Updated to address this.

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**
